projects/file-parsing/README.md

```diff
@@ -0,0 +1,56 @@
+<!--forhugo
++++
+title="File Parsing"
++++
+forhugo-->
+
+In this project we'll practice parsing data from files in different formats. Often times it's convenient to write a small program to understand or process some data, and being comfortable quickly doing so can be very helpful.
+
+Timebox: 2 days
+
+## Objectives:
+
+- Write code to parse data in non-standard formats.
```

Comment:
should we specify somewhere that we expect Go implementations?
i wonder as well: should we ask them to practice their shell by doing the same tasks using command line scripting utilities (sed/awk/jq and so on). can be a really useful skill for troubleshooting. at least for the jsons and csvs, the binary formats might be a bit much

Reply:
also maybe explicitly suggest writing unit tests, although TBF i don't think we did for the rest of the projects